src/components/ThemeProvider.tsx

```diff
@@ -0,0 +1,9 @@
+"use client";
```

Comment:
Kedže používaš pages router, tak toto je uplne zbytočné sem dávať. Tipujem že si to vykopíroval z nejakej dokumentácie, tak si iba v budúcnosti kontroluj že či je to robené na pages alebo na app router. Tu ti to možno sedí, ale väčšinou tam byvaju nejake rozdiely